Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1657,8 +1724,8 @@ public string keyboardLayout
         /// The minus ('-') key on the numpad. The key on the right side of the numpad with
         /// <see cref="numpadMultiplyKey"/> above it and <see cref="numpadPlusKey"/> below it.
         /// </summary>
-        /// <value>Control representing the numpad minus key.</value>
         /// <remarks>
+        /// <see cref="KeyControl"/> representing the numpad minus key.
```

Comment:
needs a paragraph after it?

Reply:
fixed in [8ad97b6](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/8ad97b6e1a384f347459dbfa06ba72c998464ce4)